app/build.gradle

```diff
@@ -106,19 +106,26 @@ android {
 }
 
 android.applicationVariants.all { variant ->
+
+// -------------------------------------------------------------------------------------------------
+// Set up kotlin-allopen plugin for writing tests
+// -------------------------------------------------------------------------------------------------
+
     boolean hasTest = gradle.startParameter.taskNames.find {it.contains("test") || it.contains("Test")} != null
     if (hasTest) {
         apply plugin: 'kotlin-allopen'
         allOpen {
             annotation("org.mozilla.fenix.test.Mockable")
         }
     }
-}
 
-android.applicationVariants.all { variant ->
+// -------------------------------------------------------------------------------------------------
+// Generate version codes for builds
+// -------------------------------------------------------------------------------------------------
+
     def buildType = variant.buildType.name
 
-    if (buildType == "release" || buildType == "nightly") {
+    if (buildType == "release") {
```

Comment:
Do we not need to account for the nightly buildType anymore?

Reply:
It doesn't exist. We created a separate build type for Nightly to limit the proliferation of build flavors on Focus.